.github/usdchecker.py

```diff
@@ -120,6 +120,10 @@ def main():
     warnings = checker.GetWarnings()
     errors = checker.GetErrors()
     failedChecks = checker.GetFailedChecks()
+
+    # Skip 'missing' .mtlx documents, until usd-core includes UsdMtlX
+    if failedChecks:
```

Comment:
I don't think this check is needed because the var will always an array:
https://github.com/PixarAnimationStudios/OpenUSD/blob/f941bce34505c88eaf32dc79f36eab2659b2b1ee/pxr/usd/usdUtils/complianceChecker.py#L994